azooKeyMac/InputController/SegmentsManager.swift

```diff
@@ -561,6 +564,25 @@ final class SegmentsManager {
         suggestSelectionIndex = nil
     }
 
+    /// 確定されるであろう候補を取得する
+    func getCandidateToCommit(inputState: InputState) -> Candidate {
+        let markedText = self.getCurrentMarkedText(inputState: inputState)
+        let text = markedText.reduce(into: "") { $0.append(contentsOf: $1.content) }
+
+        if let candidate = self.candidates?.first(where: { $0.text == text }) {
+            return candidate
+        }
+
+        // フォールバック：ひらがな候補を返す
+        return Candidate(
+            text: text,
+            value: 0,
+            composingCount: .inputCount(self.composingText.input.count),
+            lastMid: 0,
+            data: []
+        )
```

Comment:
好みの問題ですが、returnは1回にする方が読みやすい気がしてます
```suggestion
        return if let candidate = self.candidates?.first(where: { $0.text == text }) {
            candidate
        } else {
            // フォールバック：ひらがな候補を返す
            Candidate(
                text: text,
                value: 0,
                composingCount: .inputCount(self.composingText.input.count),
                lastMid: 0,
                data: []
            )
        }
```

Reply:
ご指摘の通り、return if を用いたコードの方が内容を把握しやすいように見えるため適用しました。
また、commitMarkedText についてもそのほうがシンプルなので反映しました。